### PR TITLE
Header

### LINKED ISSUE #17: To create a login page for users
It will be divided in 3 parts : 
- header (which will contain the name of the project and toggle button for light n dark mode )
- main body ( where the user will have the option to signup/sign in using the mail address and password )
- footer (mentioning made with love)

### PR DESCRIPTION
**Pull request by deek121477**
## Related Issue 

- Info about the related issue 

Closes: #17

### Describe the changes you've made

Give a clear description of what modifications you have made

I have added a header name of the project and a toggle button that changes to dark and light mode. All these changes are made on the Login page which consists of HTML, CSS, and JS.
## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mention any unusual behaviour of your code (Write NA if not)
NA

## Additional Info (optional)
Added login.css and login.js in separate folder.

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by ankitapuri** [comment]
image
It is displaying this only

File: login.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+</div>
+ <div class="container">
+     <div class="form">
+       <div class="heading">
+
+       </div>
+       <div class="main">
+         <div class="register">
+           <form id="register" method="post">
+           <label for="username">Name of the Project : </label>
```
Comment (avinal) [inline login.html:36]: I don't think this is the page we want, there is nothing about login. @deek121477 did you tested your changes locally??
image

Comment (avinal) [reply login.html:36]: The toggle is also not working

File: login.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+      integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+<title>Register</title>
+
+   <link href="CSS/login.css" rel="stylesheet">
```
Comment (ankitapuri) [inline login.html]: it should be css/login.css

**Comment by ankitapuri** [comment]
image
Delete this also

**Comment by deek121477** [comment]
I have made changes in html and css.login

**Comment by deek121477** [comment]
I have made changes in HTML and login.css.

**Pull request closed without merging**